### ISSUE TITLE
[Adaptive Cards - Expense Report]: Luminosity ratio of 'Reject' text is '2.733:1' which is less than 4.5:1 on Reject button under Expense Report Pane.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#96]
### Target Platforms

Android

### SDK Version

2023.12.12.1 (85454887)

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS: Android 13
Device: S20
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.12.12.1 (85454887)

**Repro Steps:**
1. Launch the Adaptive cards app on Android.
2. Navigate to 'Import' control and select it.
3. Navigate to 'Sample cards' and select it.
4. Navigate to available dropdown under 'Sample cards' and select it.
5. Navigate all the controls under dropdown and select 'Expense Report'.
6. Navigate to 'Visual' tab control and select it.
7. Navigate all the controls under 'Visual'.
8. Navigate to 'Reject' button and observe the issue with the tool.

**Actual Result:**
The Luminosity ratio of 'Reject' text is '2.733:1' which is less than 4.5:1 on Reject button under Expense Report Pane.

**Expected Result:**
Luminosity ratio of 'Reject' text should be equal or greater than 4.5:1 on Reject button under Expense Report Pane.

**User Experience:**
User with low vision might face difficulty to understand/visualize the button, If the color contrast ratio is less than 4.5:1.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![The Luminosity ratio of 'Reject' text is '2 733 1' which is less than 4 5 1 on Reject button under Expense Report Pane](https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/ca13668d-109b-44e5-92ec-1866d007a1d3)

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is now workig fine on the below build hence closing this issue.

Build 1.0.1(2)

Snippet:
![image](https://github.com/user-attachments/assets/c3680a54-f8c5-4cc5-9711-319720830086)
